Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel+FaceDetectionResultHandler.swift

```diff
@@ -28,14 +28,15 @@ extension FaceLivenessDetectionViewModel: FaceDetectionResultHandler {
             }
         case .singleFace(let face):
             var normalizedFace = normalizeFace(face)
-            normalizedFace.boundingBox = normalizedFace.boundingBoxFromLandmarks(ovalRect: ovalRect)
+            guard let sessionConfiguration = sessionConfiguration else { return }
+            normalizedFace.boundingBox = normalizedFace.boundingBoxFromLandmarks(ovalRect: ovalRect,
+                                                                                 ovalMatchChallenge: sessionConfiguration.ovalMatchChallenge)
 
             switch livenessState.state {
             case .pendingFacePreparedConfirmation:
-                if face.faceDistance <= initialFaceDistanceThreshold {
```

Comment:
is `initialFaceDistanceThreshold` a magic number that can be removed?

Reply:
yes, this is received in ovalMatchChallenge - FaceDistanceThreshold